h2o-core/src/main/java/water/fvec/Frame.java

```diff
@@ -383,7 +383,10 @@ public int find( String name ) {
   }
 
   /**   Finds the matching column index, or -1 if missing
-   *  @return the matching column index, or -1 if missing */
+   *  @return the matching column index, or -1 if missing 
+   *  @deprecated as many columns in a Frame could be backed by the same Vec, we can't return single column index that corresponds to a given {@code vec}
```

Comment:
Thank you.
Maybe just add a suggestion that the developer should modify its code to be able to use `find(String name)` instead.

Reply:
@seb-h2o yep. added suggestion.